plugins/languages/nvim-jdtls.nix

```
@@ -105,7 +105,7 @@ in {
           else cfg.data;
       in
         [
-          "${pkgs.jdt-language-server}/bin/jdt-language-server"
+          "${lib.getExe pkgs.jdt-language-server}"
```

Why use string templating here instead of an expression?

```suggestion
          (lib.getExe pkgs.jdt-language-server)
```

---

I agree, this is what I had suggested at first.
But I'm trying to stop nitpicking !

---

Sorry, I missed that in your original review. I just glanced at the suggested change from my phone and thought it was just the lib reference.

---

No worry, I fixed it :)